plugins/Editing.jsx

```diff
@@ -176,10 +177,15 @@ class Editing extends React.Component {
             );
         }
 
-        const actionButtons = [
-            {key: 'Pick', icon: 'pick', label: LocaleUtils.trmsg("editing.pick"), data: {action: 'Pick', geomReadOnly: false}},
-            {key: 'Draw', icon: 'editdraw', label: LocaleUtils.trmsg("editing.draw"), data: {action: 'Draw', feature: null, geomReadOnly: false}}
-        ];
+        const actionButtons = [];
+        if (isEmpty(editPermissions) || editPermissions.updatable === true || editPermissions.deletable === true) {
+            // Pick button will appear by default if no permissions are defined in theme editConfig or when updatable or deletable permissions are set
+            actionButtons.push({key: 'Pick', icon: 'pick', label: LocaleUtils.trmsg("editing.pick"), data: {action: 'Pick', geomReadOnly: false}});
+        }
+        if (isEmpty(editPermissions) || editPermissions.creatable === true) {
+            // Draw button will appear by default if no permissions are defined in theme editConfig or when creatable permission is set
+            actionButtons.push({key: 'Draw', icon: 'editdraw', label: LocaleUtils.trmsg("editing.draw"), data: {action: 'Draw', feature: null, geomReadOnly: false}});
+        }
```

Comment:
`editPermissions.creatable !== false` should achieve the same

Reply:
Thank you very much for your review !